core/App/ehsm_core_test.cpp

```diff
@@ -956,11 +956,14 @@ void test_GenerateQuote_and_VerifyQuote()
     printf("============test_GenerateQuote_and_VerifyQuote start==========\n");
     char challenge[32] = "challenge123456";
     char nonce[16] = "nonce123456";
-
+    // the string generated after converting the value of mr_signer and mr_enclave to hexadecimal
+    // notice: these 2 values will be changed if our enclave has been updated. then the case may be failed.
```

Comment:
you can add the cmd that i sent to you to dump the mrenclave/mrsigner as comments.

Reply:
Ok, I'll add it.